pkg/controllers/nodeclaim/disruption/controller.go

```diff
@@ -67,6 +69,7 @@ func NewController(clk clock.Clock, kubeClient client.Client, cloudProvider clou
 // Reconcile executes a control loop for the resource
 func (c *Controller) Reconcile(ctx context.Context, nodeClaim *v1.NodeClaim) (reconcile.Result, error) {
 	ctx = injection.WithControllerName(ctx, "nodeclaim.disruption")
+	ctx = log.IntoContext(ctx, log.FromContext(ctx).WithValues("Node", klog.KRef("", nodeClaim.Status.NodeName)))
```

Comment:
Same here?

Reply:
I could go either way -- yes, it could save a tiny amount of time, but I do prefer that we keep all of the context injection sort-of together at the top of the Reconcile()